pkg/networkmanager/interfaces.js

```diff
@@ -1109,7 +1115,7 @@ export function NetworkManagerModel() {
                                           'org.freedesktop.NetworkManager.Settings',
                                           'AddConnection',
                                           settings_to_nm(conf, { }))
-                        .then(path => get_object(path, type_Connection));
+                        .then(results => get_object(results[0], type_Connection));
```

Comment:
For such a case it's nice to use destructuring, and give the output arguments some descriptive name. E.g.

    .then(([con_path]) => get_object(con_path, ...))

Reply:
Good point! We could use destructuring everywhere for even better clarity.

Reply:
Done!  Unfortunately, eslint doesn't like empty array patterns like `.then(([]) => undefined)`, oh well...